src/app/_components/Map/window.js

```diff
@@ -0,0 +1,35 @@
+'use client';
+
+import { MapContainer, Marker, Popup, TileLayer } from 'react-leaflet';
+import 'leaflet/dist/leaflet.css';
+import 'leaflet-defaulticon-compatibility';
+import 'leaflet-defaulticon-compatibility/dist/leaflet-defaulticon-compatibility.css';
+import PropTypes from 'prop-types';
+
+export default function MapWindow({ points, center, zoom }) {
+  return (
+    <MapContainer center={center} zoom={zoom} scrollWheelZoom={false} style={{ height: '100%' }}>
+      <TileLayer
+        attribution='&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
+        url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
+      />
+      {points.map(({ title, latitude, longitude }) => (
+        <Marker position={[latitude, longitude]} key={title}>
+          {title && <Popup>{title}</Popup>}
```

Comment:
if a title is empty then you will return an empty `<Marker>`.
You can use `points.filter(...).map(...)` or `points.reduce(...)`

Reply:
Thanks for comment! Won't it be better just to make title prop required? In this case dev will be able to see points without title, but also see an error like this to find it in program data.
![image](https://github.com/keenethics/mayak/assets/87268303/b6087b9e-0b4f-4505-9f48-9a47cc0f696b)

Reply:
"make title prop required" is not a bad idea though. But you shouldn't rely on this, it's better to prevent potential bugs.

Reply:
Discussed in private conversation.